lib/unittest_server.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, *args, **kwargs):
     def execute(self, server):
         server.current_test = self
         execs = server.prepare_args()
+        execs[0] = os.path.abspath(execs[0])
```

Comment:
It would be nice to include a rationale into the commit message. The documentation for Python 2 and Python 3 explicitly describes the different behaviour:

From https://docs.python.org/2/library/subprocess.html#subprocess.Popen:

> If cwd is not None, the child’s current directory will be changed to cwd before it is executed. Note that this directory is not considered when searching the executable, so you can’t specify the program’s path relative to cwd.

From https://docs.python.org/3/library/subprocess.html#subprocess.Popen:

> If cwd is not None, the function changes the working directory to cwd before executing the child. <...> In particular, the function looks for executable (or for the first item in args) relative to cwd if the executable path is a relative path.

Reply:
Thanks! Added it to commit message.